config/routes.rb

```diff
@@ -23,6 +23,7 @@
   resources :issues do
     collection do
       post :import
+      delete :multiple_destroy
```

Comment:
I know I've mentioned it above, but naming convention of :actions / :routes for Issues / Notes / X